### ISSUE TITLE
Low ADAMTS13 Activity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11171]
**Preferred term label:**
Low ADAMTS13 Activity

**Synonyms**
https://www.ncbi.nlm.nih.gov/medgen/C5418984

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Thrombotic thrombocytopenic purpura, immune

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001276
- label: Decreased circulating ADAMTS13 activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:12637323
- def: Activity of the enzyme von Willebrand factor-cleaving protease (ADAMTS13) in the blood circulation below the lower limit of normal.
- comment: A severe deficiency of ADAMTS13 (a disintegrin-like and metalloprotease with thrombospondin type 1 repeats) as detected by current assays, less than 5 percent of normal activity, may be specific for Thrombotic thrombocytopenic purpura (TTP).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11171
- synonyms:
